package.json

```diff
@@ -81,6 +81,7 @@
     "nock": "^9.1.4",
     "node-mocks-http": "^1.6.5",
     "nsp": "^2.8.0",
+    "nyc": "13.0.0",
```

Comment:
Wrong version, @adr1ancho - there is no "13.0.0". Latest version (according to npm) is 12.0.2.